packages/main/src/MenuItem.hbs

```diff
@@ -4,9 +4,9 @@
 	{{#if text}}
 		<div class="ui5-menu-item-text">{{text}}</div>
 	{{/if}}
-	{{#if _additionalText}}
-		<span part="additional-text" class="ui5-li-additional-text">{{_additionalText}}</span>
-	{{/if}}
+	{{#unless hasSubmenu}}
```

Comment:
Is it necessary to render this span if there is no additional text set?

Reply:
It is not and now this empty span is removed.